sphinxcontrib/opencontracting.py

```diff
@@ -15,6 +16,7 @@
 extensions_url = 'https://raw.githubusercontent.com/open-contracting/extension_registry/main/extensions.csv'
 extension_versions_url = 'https://raw.githubusercontent.com/open-contracting/extension_registry/main/extension_versions.csv'  # noqa: E501
 extension_explorer_template = 'https://extensions.open-contracting.org/{}/extensions/{}/{}/'
+WORKED_EXAMPLES_ENV_NAME = 'worked_example_all_worked_examples'
```

Comment:
This is not an environment name. It's the name of an environment attribute.

Removing underscore to match name of main directive.

```suggestion
WORKEDEXAMPLE_ENV_ATTRIBUTE = 'workedexample_all_worked_examples'
```